app/views/coronavirus_form/hotel_rooms_number.html.erb

```diff
@@ -0,0 +1,29 @@
+<% content_for :title do %><%= t("coronavirus_form.questions.hotel_rooms_number.title") %><% end %>
+<% content_for :meta_tags do %>
+  <meta name="description" content="<%= t('coronavirus_form.questions.hotel_rooms_number.title') %>" />
+<% end %>
+
+<% content_for :back_link do %>
+  <%= render "govuk_publishing_components/components/back_link", { href: previous_path } %>
+<% end %>
+
+<%= form_tag({},
+  "data-module": "track-coronavirus-form-hotel-rooms-number",
+  "data-question-key": "hotel_rooms_number",
+  "id": "hotel_rooms_number",
```

Comment:
This should be moved to the input so that the error message link points to the right element.